esp/platform/espp.cpp

```diff
@@ -544,6 +600,9 @@ int init_main(int argc, const char* argv[])
             config->bindServer(*server.get(), *server.get());
             config->checkESPCache(*server.get());
 
+            if (fileHandler)
+                server->setFileHandler(fileHandler);
```

Comment:
this is really a logmsghandler, rename the local var, and the setFileHandler function